modules/dxp/apps/portal-search-tuning/portal-search-tuning-blueprints-admin-web/src/main/resources/META-INF/resources/css/_preview_sidebar_modal.scss

```diff
@@ -0,0 +1,5 @@
+.preview-sidebar-modal {
+	.btn-group {
+		padding-bottom: 12px;
+	}
+}
```

Comment:
CSS files should map to the component name. Since `preview-sidebar-modal` is part of the PreviewSidebar component, let's put this in `_preview_sidebar.scss`. This'll make it easier to find which files affect which components.

Reply:
Oh yea I thought it was a bit much to make a whole file for one property haha, but is that how you apply it for a modal? It's not under `.portlet-blueprint-admin`'s class?

Reply:
Ah right, modal css needs to be top-level. Can we rename this class as if it was an independent component? 

Or better yet, maybe adding the class name to the `<ClayModal>` in `PreviewModal.js` since I could see this being useful across all PreviewModals. Maybe `preview-modal` so it's related to the component.

Some food for thought: It'd be more informational to name them related to the component if it's a shared component, but in the case it's a single instance, we should name it based on the content. So if this was a class inside of `PreviewSidebar.js` a class `raw-response-modal` is much more helpful.